src/test/debuginfo/msvc-pretty-enums.rs

```diff
@@ -49,14 +49,30 @@
 // cdb-check:        [+0x000] __0              : 0x0 [Type: unsigned int *]
 // cdb-check:    [+0x000] discriminant     : None (0x0) [Type: enum$<core::option::Option<u32*>, 1, [...], Some>::Discriminant$]
 
+// cdb-command: dx -r2 h,!
+// cdb-check:h,!              : Some [Type: enum$<core::option::Option<u32>>]
+// cdb-check:    [+0x000] variant0         [Type: enum$<core::option::Option<u32>>::None]
+// cdb-check:    [+0x000] variant1         [Type: enum$<core::option::Option<u32>>::Some]
+// cdb-check:        [+0x004] __0              : 0xc [Type: unsigned int]
+// cdb-check:    [+0x000] discriminant     : Some (0x1) [Type: core::option::Option]
```

Comment:
Does the `-r2` flag disable natvis? It would be great to give a minimal description of how the `dx` command is used here at the top of the file.

Reply:
No, the `-r2` flag means "expand **r**ecursively 2 levels". The [`!` format specifier](https://docs.microsoft.com/en-us/visualstudio/debugger/format-specifiers-in-cpp?view=vs-2019#BKMK_Visual_Studio_2012_format_specifiers) is what disables the natvis for this expression.